frontend/src/libs/components/order-card/order-card.tsx

```diff
@@ -0,0 +1,104 @@
+import { PlainSvgIconName } from '~/libs/enums/enums.js';
+import { getValidClassNames } from '~/libs/helpers/helpers.js';
+
+import { PlainSvgIcon } from '../plain-svg-icon/plain-svg-icon.jsx';
+import style from './styles.module.scss';
+
+type Properties = {
+  driver: {
+    firstName: string;
+    lastName: string;
+    profileURL: string;
+  };
```

Comment:
maybe let's create separate type for it called for example DriverProfileInfo or something like this?